src/main/java/com/wcg/liquibase/config/rules/RuleConfig.java

```diff
@@ -13,7 +13,7 @@
 
     private static final String DYNAMIC_VALUE = "{{value}}";
 
-    private final boolean enabled;
+    private final Boolean enabled;
```

Comment:
I was going to question if there is a use case for this, but I guess it's being able to switch off a rule (maybe on a temporary basis) without losing your configuration of it.

Using an object Boolean with null meaning true does feel a little off though - what about if we kept it as a `boolean`, but initialised it as `true` in the builder?

Reply:
I agree this is a bit gross. Have changed to default the enabled to true in the builder instead.